nixos/modules/security/sudo.nix

```diff
@@ -216,9 +225,20 @@ in
         ${cfg.extraConfig}
       '';
 
-    security.wrappers = {
-      sudo.source = "${cfg.package.out}/bin/sudo";
-      sudoedit.source = "${cfg.package.out}/bin/sudoedit";
+    security.wrappers = let
+      owner = "root";
+      group = if cfg.execWheelOnly then "wheel" else "root";
```

Comment:
Uhm, `owner = "root"`, setuid, etc. seem to be the default already why are you setting them?
Also, I would have suggested to use a `mkIf cfg.execWheelOnly`, but  I'm not sure it works here:
I looked at the `security.wrappers` module and found out (quite shockingly) that it's not a submodule but untyped `attrs`. This is bad for usability and also makes the module not documented in the options manual section. This is going to be the next thing I'll work on.

Reply:
> Also, I would have suggested to use a `mkIf cfg.execWheelOnly`, but I'm not sure it works here:

That's what I tried first, didn't work.

> Uhm, `owner = "root"`, setuid, etc. seem to be the default already, why are you setting them?

Because if `permissions` is set, the other defaults no longer apply:
https://github.com/NixOS/nixpkgs/blob/eadffd915463ecbc93f7dc856b7059bd816e6b6a/nixos/modules/security/wrappers/default.nix#L67-L86

> I looked at the `security.wrappers` module and found out (quite shockingly) that it's not a submodule but untyped `attrs`. This is bad for usability and also makes the module not documented in the options manual section. This is going to be the next thing I'll work on.

Ping me in that pull request and I'll be happy to review it!